### ISSUE TITLE
0.10.7 Windows 10 Can't load saved jersey

### ISSUE THREAD

**Issue by EastonThewes** [eriqjaffe/OOTP-Uniform-Maker#81]
Whenever I try to load in a saved jersey, it just gets stuck and won't import it.

**Comment by eriqjaffe** [comment]
Loading has always been problematic.  Just to be sure - this is a uniform created with 0.10.7?

**Comment by EastonThewes** [comment]
Yeah, I created it with 0.10.7. Seems like the fields are loading, but rendering the jersey never completes. This was with the portal exe. I cloned the repo and tried to drill into it some and seems to me like its getting hung up on line 17227, the jerseyBelow.loadFromJSON. I haven't spent a lot of time with it so I'm not 100% for sure.

**Comment by eriqjaffe** [comment]
I found the issue.  Guess fabric.js didn't like the vintage panel groups having the "type: 'vintageGroup'" property.  Changed the way they get shown and hidden, removed the type property from the groups and now things work again.  :D

**Comment by eriqjaffe** [comment]
I found the issue.  Guess fabric.js didn't like the vintage panel groups having the "type: 'vintageGroup'" property.  Changed the way they get shown and hidden, removed the type property from the groups and now things work again.  :D

**Comment by EastonThewes** [comment]
You're a legend. Verified working on my end. Thank you!
